### DISCUSSION TITLE
Vietnamese language pls!

### DISCUSSION THREAD

**Discussion by VHDung3011** [YaoFANGUK/video-subtitle-extractor#210] [Q&A] [answered]
Can you add Vietnamese subtitles?
This is the Vietnamese corpus: https://github.com/duyvuleo/VNTC
Thank you!

**Comment by trunghuyne** [comment]
We hope the Admin can provide further support for Vietnamese language recognition. Thank you very much.

**Comment by YaoFANGUK** [comment, marked as answer]
supported but no accents in subtitles

**Reply by VHDung3011** [reply]
thank you a lot~!

**Reply by vcstack** [reply]
@YaoFANGUK Thank you, can you help me with accents in Vietnamese?
